lib/galaxy/datatypes/binary.py

```diff
@@ -855,11 +860,15 @@ def set_meta(self, dataset, overwrite=True, **kwd):
                 dataset.metadata.col_attrs_count = len(tmp)
                 dataset.metadata.col_attrs_names = tmp
 
-                tmp = list(loom_file['col_graphs'].keys())
+                # According to the Loom file format specification, col_graphs
+                # and row_graphs are mandatory groups, but files created by
+                # Bioconductor LoomExperiment do not always have them:
+                # https://github.com/Bioconductor/LoomExperiment/issues/7
+                tmp = list(loom_file.get('col_graphs', {}).keys())
```

Comment:
But we should stick to the specification here and hack around this in the tools?

Reply:
This PR prevents an unnecessary warning in the dataset info, these metadata elements would be empty any way.
We don't have control on how Loom dataset are created using Bioconductor LoomExperiment (could be outside Galaxy or in any number of Galaxy tools), the correct solution is to fix https://github.com/Bioconductor/LoomExperiment/issues/7 .